src/Compilers/Core/Portable/Symbols/ILocalSymbol.cs

```diff
@@ -23,11 +23,17 @@ public interface ILocalSymbol : ISymbol
         bool IsConst { get; }
 
         /// <summary>
-        /// Returns true if this local is a ref-local.
+        /// Returns true if this local is a ref local or a ref readonly local.
+        /// Use <see cref="RefKind"/> to get more detailed information.
         /// </summary>
         bool IsRef { get; }
```

Comment:
Why keep this property if the same information is in RefKind?

Reply:
> Why keep this property if the same information is in RefKind?

We already shipped it, cannot take it back.